opensrp-chw/src/kk/assets/json.form/kk_anc_pregnancy_outcome.json

```diff
@@ -691,6 +691,10 @@
               "rules-file": "kk_anc_pregnancy_outcome_relevance.yml"
             }
           }
+        },
+        "v_required": {
+          "value": "true",
+          "err": "{{anc_pregnancy_outcome.step2.caregiver_surname.v_required.err}}"
```

Comment:
This should have its own string resource, instead of using caregiver_surname, it might confused the user if the text is for surname